.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "i18n"]
 	path = i18n
-	url = https://github.com/ourjapanlife/findadoc-localization
+	url = https://github.com/2moon/findadoc-localization.git
```

Comment:
Can you please remove this from the Pull Request?

Reply:
done